app/views/users/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="user">
+	<div class="user-img">
+		<%= image_tag("user.png")%>
+	</div>
+	<div class="user-text border">
+		<h2 class="user-heading"><%= link_to user.Name, "/users/#{user.id}"%></h2>
```

Comment:
- [ ] As requested previously, you are yet to fully set up your routing and navigations the Rails way.
 
  When you run `rails routes` in your terminal, you can see the paths for your routes. Under the **Prefix** column heading, you can find different ways to define your links. I would recommend you use the `_path` method as provided by Rails. Instead of directing defining the paths, you could have something like this:

  ```
  <h2 class="user-heading"><%= link_to user.Name,  user_path(user) %></h2>
  ```
  OR
  ```
  <h2 class="user-heading"><%= link_to user.Name, user %></h2>    #=> Rails can decipher what to link to
  ```